docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -75,6 +75,28 @@ General-purpose aggregation functions
     Returns ``True``, if all input values are true, ``None`` if all values are
     null or if there are no values, otherwise ``False`` .
 
+    Example:
+    	
+	Assume ``some_boolean_field`` is a non-nullable BooleanField and
+	there exists at least one instance of ``SomeModel`` where
+	``some_boolean_field = False``:
+        
+	>>> SomeModel.objects.aggregate(booland=BoolAnd('some_bool_field'))
+	{'booland': False}
+
+
+    BoolAnd can be used with :ref:`Q objects<complex-lookups-with-q>`.
+
+    Example:
+
+	Assume ``some_int_field`` is a non-nullable IntegerField and there
+	exists at least one instance of ``SomeModel`` where ``some_int_field``
+	is a value of 2 or less:
+
+	>>> SomeModel.objects.aggregate(booland=BoolAnd(Q('some_int_field__gt=2')))
+	{'booland': False}
```

Comment:
I would change examples to more explicit (like in [Database functions docs](https://docs.djangoproject.com/en/3.0/ref/models/database-functions/)):
 ```suggestion
    Usage example::
        
        class Comment(models.Model):
            body = models.TextField()
            published = models.BooleanField()
            rank = models.IntegerField()
        
        >>> from django.contrib.postgres.aggregates import BitAnd
        >>> Comment.objects.aggregate(boolor=BoolAnd('published')
        {'boolor': False}
        >>> Comment.objects.aggregate(boolor=BoolAnd(Q(rank__lt=100)))
        {'boolor': True}
```

Reply:
Thanks for the link.  I'll use that as a reference from now on.